Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -179,6 +185,7 @@ public struct Configuration: Equatable {
             }
             self.init(dict: dict)!
             configurationPath = fullPath
+            hash = dict.bridge().hashValue
```

Comment:
This is returning a value of `3` for Realm. Seems suspicious: https://github.com/realm/realm-cocoa/blob/master/.swiftlint.yml